BEIMA.Client/src/pages/Devices/DevicePage.js

```diff
@@ -248,11 +226,17 @@ const DevicePage = () => {
       const delDocs = removedDocs
 
       // Hit endpoints here
-      console.log(newDevice)
+      updateDevice(newDevice);
       console.log(newImage)
```

Comment:
Do we still need these logs?

Reply:
I left them in there since those api calls still need to be added. And the build process with be angry if we have unused variables